javalin/src/test/java/io/javalin/TestBodyReading.kt

```diff
@@ -75,9 +75,9 @@ class TestBodyReading {
                 ctx.result(formParamString)
             }
         }
-        val params = "a=1&a=2&a=3&b=1&b=2&c=1&d=&e&f=%28%23%29"
+        val params = "a=1&a=2&a=3&b=1&b=2&c=1&d=&e&%28+f+%29=%28%23%29"
```

Comment:
Could you use `URLEncoder` to encode this, and maybe add a new one at the end instead of changing an existing one?

Reply:
I guess past @tipsy didn't follow his own advice, could you update the existing ones to use `URLEncoder` too ? 😬

Reply:
I updated the test and cleaned it up a bit.

Reply:
Looks great, thanks for cleaning it up!